classes/maps.php

```diff
@@ -212,13 +207,13 @@ function generate_iframe() {
 				<html style='margin-top:0 !important;padding-top:0 !important;'>
 				<head>
 					<?php wp_head(); ?>
-					<style type='text/css'>* {
+					<style type='text/css'>
+						body {
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->